src/components/dashboard/roaster/subscriptions/RoasterItem.js

```diff
@@ -0,0 +1,47 @@
+import React, {Component, PropTypes} from 'react';
+import RoasterSubscription from './RoasterSubscription';
+import BeanItemImage from '../../browse/BeanItemImage';
+import Loading from '../../../Loading';
+
+class RoasterItem extends Component {
+    render() {
+        const subs = this.props.subscriptions;
+
+        if(!subs) {
+            return (<Loading fetching={true} />);
+        }
+
+        return (
+            <div className="center ba pa4 w-100 overflow-y-auto">
+                <div className="flex">
+                    <div className="pl3 flex-column w-40" style={{maxWidth: '75px'}}>
+                        <BeanItemImage src={this.props.bean.pictureURL} alt={this.props.bean.name}/>
+                    </div>
+                    <div className="pl3 flex flex-column justify-between w-60">
+                        <div className="f1 b">{this.props.bean.name}</div>
+                    </div>
+                    <div className="pl3 flex-column w-50">
```

Comment:
how does this look when we scale down to mobile-sized screens? we can use the tachyons media tags if need be.